content/administration/install/deploy.rst

```diff
@@ -388,20 +388,109 @@ The solutions to support livechat/motifications in a WSGI application are:
   with ``/longpolling/`` to
   :option:`the longpolling port <odoo-bin --longpolling-port>`.
 
+.. _streaming:
+
+Serving Static Files and Attachments
+====================================
+
+For development convenience, Odoo directly serves all static files and attachments in its modules.
+This may not be ideal when it comes to performances, and static files should generally be served by
+a static HTTP server.
+
 Serving Static Files
-====================
+--------------------
+
+Odoo static files live in each module's ``static/`` folder, so static files can be served by
+intercepting all requests to :samp:`/{MODULE}/static/{FILE}`, and looking up the right module
+(and file) in the various addons paths.
+
+Say Odoo has been installed via the **debian packages** for community and enterprise, the addons
```

Comment:
- Is everything that follows an example? You could use the example admonition to mark it as such
https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#contributing-example
- "community and enterprise" are proper nouns and must be capitalized. 
```suggestion
Say Odoo has been installed via the **debian packages** for Community and Enterprise, the addons
```